pkg/webhooks/provisioning/v1alpha1/defaulter.go

```diff
@@ -49,20 +53,39 @@ func (v *Defaulter) Handle(ctx context.Context, req admission.Request) admission
 	if err != nil {
 		return admission.Errored(http.StatusBadRequest, err)
 	}
-	v.applyDefaults(&provisioner.Spec)
+	if err := v.applyDefaults(ctx, &provisioner.Spec); err != nil {
+		return admission.Errored(0, fmt.Errorf("applying defaults to provisioner, %w", err))
```

Comment:
Can you use a canonical http status code here?

Reply:
ha yeah... 🙈